rust/core-lib/src/lib.rs

```diff
@@ -24,19 +24,19 @@ extern crate syntect;
 extern crate toml;
 extern crate notify;
 
-#[cfg(target_os = "fuchsia")]
```

Comment:
You might be able to save yourself some work here by doing something like this:
```rust
#[cfg(feature = "ledger")]
mod ledger_includes {
    pub extern crate fuchsia_zircon;
    pub extern crate fuchsia_zircon_sys;
    pub extern crate mxruntime;
    pub extern crate fidl;
    ....
}
#[cfg(feature = "ledger")]
use ledger_includes::*;
```

Reply:
Good call, I'll do that.